### PR TITLE
Statusjulia

### PR DESCRIPTION
**Pull request by julezjw**
Created 'c-status' command in c-panel that allows user to see access status for every existing  conversation
There are a couple simple tests that check very basic functionalities

### CODE REVIEW

File: src/codeu/chat/client/commandline/Chat.java [hunk 1]
```diff
@@ -554,6 +557,28 @@ public void invoke(List<String> args) {
...
+		// every conversation they are currently a part of  
+		panel.register("c-status", new Panel.Command() { 
+			@Override
+			public void invoke(List<String> args) { 
+				// HashMap stores the name of every convo the user is a part of and the access level
+				// for that conversation
+				HashMap<String, UserType> convoAccess = new HashMap<String, UserType>();  
+				// adds all convos to convo access with name of convo as key 
+				// and user's access status as value 
+				for (ConversationContext convo : user.conversations()){
```
Comment (KevinWorkman) [inline src/codeu/chat/client/commandline/Chat.java:572]: Google's favorite question: what if the user has a million conversations?

(Not saying you have to change anything here. Just trying to get you thinking about scalability!)

File: test/codeu/chat/common/CstatusTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+public class CstatusTest {
```
Comment (KevinWorkman) [inline test/codeu/chat/common/CstatusTest.java:13]: Awesome to see tests. Great start! You might also think about handling error cases: is it possible for a user to enter bad data? What happens if you request the access of a non-member? Try to think about corner cases, not just the [happy path](https://en.wikipedia.org/wiki/Happy_path).

File: test/codeu/chat/common/CstatusTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    @Test
+    public void testGetAccessOf() {
+        ConversationHeader convo = new ConversationHeader(new Uuid(10), new Uuid(15), Time.now(), "Title"); 
+		convo.getAccessOf(new Uuid(10)); 
+		assertEquals(UserType.MEMBER, convo.getAccessOf(new Uuid(10)));
+    }
+    
+    @Test
+    public void TestToString() {
```
Comment (KevinWorkman) [inline test/codeu/chat/common/CstatusTest.java:23]: nit: This should start with a lower-case letter, since it's a function.

**Review by KevinWorkman — APPROVED** [review]
Nice work. Couple comments below. But overall I'll say this LGTM!

**Review by Priya411 — APPROVED** [review]
